common-utils/src/main/java/eu/arrowhead/common/SystemInfo.java

```diff
@@ -42,6 +42,18 @@ public abstract class SystemInfo {
 	@Value(Constants.$AUTHENTICATION_POLICY_WD)
 	private AuthenticationPolicy authenticationPolicy;
 
+	@Value(Constants.$MQTT_API_ENABLED_WD)
+	private boolean mqttEnabled;
+
+	@Value(Constants.$MQTT_BROKER_ADDRESS_WD)
+	private String mqttBrokerAddress;
```

Comment:
in the init() method you should check if MQQT enabled then this address is not null or empty